test/stream/api/source/version-detail-404/003-version-detail/expected-response.json

```diff
@@ -0,0 +1,5 @@
+{
+  "statusCode": 404,
+  "error": "stream.api.sourceVersionNotFound",
+  "message": "Source version \"my-source/0000000025\" not found in the branch."
+}
```

Comment:
Missing new lines at the end of some files...

Reply:
47f6964